core/src/main/java/com/linecorp/armeria/server/HttpServerPipelineConfigurator.java

```diff
@@ -461,7 +471,8 @@ private void addHttpHandlers(ChannelHandlerContext ctx) {
                     config.http1MaxHeaderSize(),
                     config.http1MaxChunkSize()));
             p.addLast(new Http1RequestDecoder(config, ch, SCHEME_HTTPS, writer));
-            p.addLast(new HttpServerHandler(config, gracefulShutdownSupport, writer, H1, proxiedAddresses));
+            p.addLast(new HttpServerHandler(configHolder,
+                    gracefulShutdownSupport, writer, H1, proxiedAddresses));
```

Comment:
nit: indent?

Reply:
I used intellij to reformat these lines of code. I have configured my ide to use Line corps settings.jar. Not sure what is the problem.